CHANGELOG.md

```diff
@@ -69,6 +69,9 @@ This project tries to follow [SemVer 2.0.0](https://semver.org/).
 - Added support for the TZ environment variable (setting timezones ex.
   `"Europe/Stockholm"`) through the tzdata package. (#20)
 
+- Removed `internal/httputils`, which was moved to
+  `github.com/iver-wharf/wharf-core/pkg/cacertutil`. (#28)
+
```

Comment:
v2.0.0 has been released. Remember to add a new WIP version section to the CHANGELOG.md